parcel/ParcelDistributions.jl

```diff
@@ -54,3 +57,27 @@ function distribution_moments(::Gamma, qₗ, Nₗ, ρₗ, ρ_air, qᵢ, Nᵢ, ρ
     end
     return (; rₗ, Aₗ, Vₗ, rᵢ)
 end
+
+function distribution_moments(::Lognormal, qₗ, Nₗ, ρₗ, ρ_air, qᵢ, Nᵢ, ρᵢ, distr_params)
+    (; σ_g) = distr_params
+    # liquid droplets
+    if Nₗ == FT(0)
+        rₗ = FT(0)
+        Aₗ = FT(0)
+        Vₗ = FT(0)
+    else
+        r_median = cbrt(3 / 4 / FT(π) / Nₗ * qₗ / ρₗ * ρ_air / exp(9 / 2 * (log(σ_g))^2))
+        rₗ = r_median * exp(0.5 * (log(σ_g))^2)
+        Aₗ = 4 * FT(π) * r_median^2 * exp(2 * (log(σ_g))^2)
+        Vₗ = 4 / 3 * FT(π) * r_median^3 * exp(9 / 2 * (log(σ_g))^2)
+    end
+    # ice crystals; note this is gamma distribution
```

Comment:
This is definitely confusing. If we want to have different distributions for both liquid and ice and we should split the `distribution_moments` function to return things for one phase only. And then call it twice in the code for liquid and ice. And make sure to throw an error if someone tries to have Lognormal distribution for ice